framework/src/multiapps/TransientMultiApp.C

```diff
@@ -395,8 +445,61 @@ TransientMultiApp::solveStep(Real dt, Real target_time, bool auto_advance)
             }
           }
         }
-        else if (!ex->lastSolveConverged())
-          throw MultiAppSolveFailure(name() + " failed to converge");
+        else
+        {
+          if (!ex->lastSolveConverged())
+          {
+            // Even if we don't allow auto_advance - we can still catch up to the current time if
+            // possible
+            if (_catch_up)
+            {
+              _console << "Starting Catch Up!" << std::endl;
+
+              bool caught_up = false;
+
+              unsigned int catch_up_step = 0;
+
+              Real catch_up_dt = dt / 2;
+
+              while (!caught_up && catch_up_step < _max_catch_up_steps)
+              {
+                Moose::err << "Solving " << name() << " catch up step " << catch_up_step
```

Comment:
I know we discussed this yesterday. It'll be nice to come up with a better solution in the future rather than printing to a different stream.

Reply:
I'm going to try switching this to `_console`